core/base/trackingFromCriticalPoints/TrackingFromCriticalPoints.cpp

```diff
@@ -169,6 +169,24 @@ void ttk::TrackingFromCriticalPoints::performMatchings(
                       sad_2Map[i], minMap[i]);
   }
 
+  int n_max = std::accumulate(
+    maxScalar.begin(), maxScalar.end(), 0,
+    [](int sum, const std::vector<double> &v) { return sum + v.size(); });
+  int n_sad_1 = std::accumulate(
+    sad_1Scalar.begin(), sad_1Scalar.end(), 0,
+    [](int sum, const std::vector<double> &v) { return sum + v.size(); });
+  int n_sad_2 = std::accumulate(
+    sad_2Scalar.begin(), sad_2Scalar.end(), 0,
+    [](int sum, const std::vector<double> &v) { return sum + v.size(); });
+  int n_min = std::accumulate(
+    minScalar.begin(), minScalar.end(), 0,
+    [](int sum, const std::vector<double> &v) { return sum + v.size(); });
+
+  this->printMsg("Processing " + std::to_string(n_max) + " maximas");
```

Comment:
please present `printMsg` messages in the following order:
- minima (not "minimas")
- 1-saddles (not "1_saddles")
- 2-saddles (not "2_saddles")
- maxima (not "maxima")